src/model/Examples.ts

```diff
@@ -52,6 +53,7 @@ export class Examples {
           title: 'Extract images from PDF',
         },
         {id: FeatureId.ViewPages, title: 'View Image Results'},
+        {id: FeatureId.FinderDocumentScanner, title: 'Finder Document Scanner'},
```

Comment:
I'd move this right below the Document Scanner feature and call it "Document Scanner with Finder"